dealer.py

```diff
@@ -0,0 +1,51 @@
+"""Dealer Class"""
+"""Dealer collaborates with Hand, Shoe, and Game"""
+"""Dealer is responsible for having a hand with a value and taking a turn."""
+
+from hand import Hand
+from shoe import Shoe
+# from player import Player
+from card import Card
+
+
+class Dealer():
+
+    def __init__(self):
+        self.dealer_hand = Hand()
+
+
+    def __str__(self):
+        return "The dealer has {}".format(self.dealer_hand)
+
+
+    def dealer_turn(player_hand_value):
```

Comment:
This method could be simplified in many ways. The `while True` loop is unnecessary, and a lot of the logic is duplicated. I would move the one path that doesn't return False to the top and then turn all the `return False` statements into one at the end.